src/oracc_cmd.ts

```diff
@@ -0,0 +1,29 @@
+import * as child_process from 'child_process';
+import { homedir } from 'os';
+import * as process from 'process';
+import { promisify } from 'util';
+import * as vscode from 'vscode';
+
+const execFile = promisify(child_process.execFile);
+
+function run_cmd(file: string,
+                 args: string[] = [],
+                 options: Object = {
+                     cwd: homedir(),
+                     env: {
+                         ...process.env,
+                         ...{PATH: '/usr/local/oracc/bin:' + process.env.PATH},
+                     },
+                 }) {
+    return execFile(file, args, options);
+}
+
+export async function run_oracc(action: string) {
+    try {
+        const { stdout } = await run_cmd('oracc', [action]);
+        vscode.window.showWarningMessage(`${stdout}`);
```

Comment:
This is really just a placeholder, I don't think the Emacs plugin does anything special with the stdout of the commands, I don't know what we should do.

Reply:
Maybe log to the console or file? (Would the output be useful to users or developers for debugging?)
And show a success info message instead, like we do for validation/lemmatisation?